pallets/loans/src/types.rs

```diff
@@ -86,15 +86,16 @@ pub struct WriteOffGroupInput<Rate> {
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub enum WriteOffStatus<Rate> {
 	None,
-	WrittenOff {
-		/// write off group index in the vec of write off groups
-		write_off_index: u32,
+	WrittenDownByPolicy {
+		/// percentage of outstanding debt we are going to write down on the loan
+		percentage: Rate,
+		/// additional interest that accrues on the written down loan as penalty
+		penalty_interest_rate_per_sec: Rate,
 	},
-	// an admin can write off an asset to specific percentage and penalty rate
-	WrittenOffByAdmin {
-		/// percentage of outstanding debt we are going to write off on a loan
+	WrittenDownByAdmin {
+		/// percentage of outstanding debt we are going to write down on the loan
 		percentage: Rate,
-		/// additional interest that accrues on the written off loan as penalty
+		/// additional interest that accrues on the written down loan as penalty
 		penalty_interest_rate_per_sec: Rate,
 	},
 }
```

Comment:
In the issue, @Frederik also specifies a `WrittenOff` state. I understand we still want to express this state explicitly, right?.

Reply:
Yeah good point, `WrittenOff` state is missing here.